### ISSUE TITLE
Custom token delimiter

### ISSUE THREAD

**Issue by ejurgelionis** [Pictela/gulp-token-replace#9]
Hi there,

I know it might sound a bit strange, but ability to define your own delimiter of token instead of "." would be great.

Let me give you an example.

I have an object of tokens which represents a folder structure.

```json
{"my_first_folder": {"my_first_file.html": "<content>"}, "my_second_file.html": "<content>"}
```

If I am trying to use the token `{{my_first_folder.my_first_file.html}}` it fails (due to ".html"). If I be able to define my own delimiter "/", the usage `{{my_first_folder/my_first_file.html}}` would be both, more natural and correct.

Thanks!

P.S. I am willing to make a contribution and write this by myself.

**Comment by pictelaUK** [comment]
That sounds great feel free to contribute and let me know if you need any assistance or info about the current code / test structure. 

Best,
Martin

**Comment by ejurgelionis** [comment]
@pictelaUK

I have made some changes and created a pull request https://github.com/Pictela/gulp-token-replace/pull/10.

Please review the changes and notify me if any other modifications are required.

**Comment by pictelaUK** [comment]
Released (1.0.3). Many thanks for the contribution!
